src/gui/editors/AutomationEditor.cpp

```diff
@@ -2095,7 +2052,7 @@ AutomationEditorWindow::AutomationEditorWindow() :
 	editActionsToolBar->addAction(m_flipYAction);
 
 	// Interpolation actions
-	DropToolBar *interpolationActionsToolBar = addDropToolBarToTop(tr("Interpolation controls"));
+	m_editor->m_interpolationActionsToolBar = addDropToolBarToTop(tr("Interpolation controls"));
```

Comment:
You could perhaps do something like this instead of hiding the toolbar
```cpp
connect(m_editor, &AutomationEditor::currentClipChanged, this, [this]{ setEnabled(m_editor->hasClip()); });
```